build.sbt

```diff
@@ -40,6 +40,7 @@ lazy val kafkaLagExporter =
         Testcontainers,
         AkkaHttp
       ),
+      libraryDependencies += "com.lightbend.akka" %% "akka-stream-alpakka-influxdb" % "1.1.2",
```

Comment:
Should follow the same pattern as other dependencies (see `project/Dependencies.scala`)